frontends/infinite-corridor/src/scss/fieldpage.scss

```diff
@@ -131,7 +131,13 @@ $avatar-dimension-small: 22px;
 
 
 $minTabHeigt: 44px;
```

Comment:
Basically all the CSS changes except this file are moving card-styling so that the affects all cards (e.g., widgets) rather than just learning resource cards.